src/ngraph/frontend/onnx_import/utils/matmul_factory.cpp

```diff
@@ -174,34 +174,77 @@ std::shared_ptr<ngraph::Node>
     QLinearMatmulFactory::make_dot(const std::shared_ptr<ngraph::Node>& left,
                                    const std::shared_ptr<ngraph::Node>& right)
 {
```

Comment:
Change these to Output<Node>

Reply:
Thats onnx bridge code which hasn't been touched in this PR. The change you are mentioning needs to happen at a lot of other places as well in onnx bridge.